docs/Makefile

```diff
@@ -1,3 +1,19 @@
+########################################################################################################################
+##### CUSTOM CONFIGURATION
+VENV = env/bin/activate
+PORT = 8000
+# CORE COMMANDS
+install:
+	##### setup virtualenv
```

Comment:
The tab character is a very important part of the syntax of Makefiles. All command lines must start with tabs.
That's why we have this line in `.editorconfig` https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/.editorconfig#L15